exercises/G-numbers/exercise.js

```diff
@@ -1 +1,8 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 27;
+let numberOfMentors = 20;
+console.log(
+  `Number of students: ${numberOfStudents}
+Number of mentors: ${numberOfMentors}
+Total number of students and mentors: ${numberOfMentors+numberOfStudents}`
+);
```

Comment:
Since you are not changing the number variables at any point they should be stored as const not let

Reply:
Done 👍 Thank You